assignments/array-methods.js

```diff
@@ -54,30 +57,74 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
+//Combine both the first and last names into a new array called fullName. 
+let fullName = runners.forEach(function(currentValue){
+    console.log(`${currentValue.first_name} ${currentValue.last_name}`);
+});
+console.log(fullName );
+
+
 
 // ==== Challenge 2: Use .map() ====
-// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
-console.log(allCaps); 
+// The event director needs to have all the runner's first names converted to uppercase because the director
+// BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
+let allCaps = runners.map(currentValue=>{
+    console.log(currentValue.first_name.toUpperCase());
+});
+console.log(allCaps);
+
 
 // ==== Challenge 3: Use .filter() ====
-// The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
+// The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with 
+//large sized shirts so they can choose a different size. Return an array named largeShirts that contains 
+//information about the runners that have a shirt size of L and log the result
+let largeShirts = runners.filter(currentValue=>{
+    return (currentValue.shirt_size==="L");
+});
 console.log(largeShirts);
 
+
+
 // ==== Challenge 4: Use .reduce() ====
-// The donations need to be tallied up and reported for tax purposes. Add up all the donations into a ticketPriceTotal array and log the result
-let ticketPriceTotal = [];
+// The donations need to be tallied up and reported for tax purposes. Add up all the donations into a 
+//ticketPriceTotal array and log the result
+let ticketPriceTotal = runners.reduce((total,currentValue)=>{
+return (total+currentValue.donation);
+},0);
 console.log(ticketPriceTotal);
 
+
+
 // ==== Challenge 5: Be Creative ====
-// Now that you have used .forEach(), .map(), .filter(), and .reduce().  I want you to think of potential problems you could solve given the data set and the 5k fun run theme.  Try to create and then solve 3 unique problems using one or many of the array methods listed above.
+// Now that you have used .forEach(), .map(), .filter(), and .reduce().  I want you to think of potential 
+//problems you could solve given the data set and the 5k fun run theme.  Try to create and then solve 3 unique 
+//problems using one or many of the array methods listed above.
 
 // Problem 1
+// everyone increased their donations by 20%
+
+let increaseDonations=runners.map(currentValue=>{
+    return(currentValue.donation+currentValue.donation*.2);
+});
+console.log(increaseDonations);
```

Comment:
You can simply your returned expression further like so: `currentValue.donation * 1.2` (does the same thing as what you had, but without a need to use `+`).